### PR TITLE
Select all

### LINKED ISSUE #105: Behavior to select all elements
## Problem
Related to renew-js/renew-ui#16 there should be a behavior that selects all elements.

### PR DESCRIPTION
**Pull request by tim-kilian**
Closes #105 

## Describe your changes
- Extended selection provider by a `selectAll` method and the element registery
- Added a `select.all` behavio
- Added a test for to check if all elements got selected
- Removed handles from the `getAll` method of the `elementRegistry`

### CODE REVIEW

File: src/core/ElementRegistry.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    constructor (props) {
+        super(props);
+    }
+
+    /**
+     * TODO: Remove hotfix for handles
+     * @return {*}[]
+     */
+    getAll () {
+        return super.getAll().filter((element) => element.type !== 'handle');
```
Comment (0x4d) [inline src/core/ElementRegistry.js:15]: Good hotfix :grin: Improved behavior of handles (#99) should best be implemented soon.

**Review by 0x4d — APPROVED** [review]
LGTM

File: src/core/ElementRegistry.js (hunk 1 above)
Comment (tim-kilian) [reply src/core/ElementRegistry.js:15]: Thanks :D We interchanged that this will be your task 🙃. I've assigned you in the issue #99.

Comment (0x4d) [reply src/core/ElementRegistry.js:15]: :+1:

**Pull request merged**